README.md

```diff
@@ -79,26 +79,38 @@ checking the output images.
 Options
 -------
 
-The ``@pytest.mark.mpl_image_compare`` marker can take an argument which is the
-RMS tolerance (which defaults to 2):
+Tolerance
+^^^^^^^^^
+
+The RMS tolerance for the image comparison (which defaults to 2) can be
```

Comment:
I'll do this in a separate pull request, since I think it may cause issues when running the CI with Matplotlib 1.4. If it works with 1.5 and 2.0 I'm inclined to wait until 2.0 final is out, then drop support for 1.4 in pytest-mpl.